File: src/main/java/de/domradio/domain/OnAir.java
package de.domradio.domain;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root
public class OnAir {
    @Element
    public String title;
    @Element
    public String artist;
    @Element
    public String type;
    @Element
    public String presenter;
    @Element
    public String date;
    @Element
    public String duration;
    @Element
    public String start;
    @Element
    public String cover;
    @Element
    public String TitelID;
    @Element
    public String ElementID;
    @Element
    public String ArchivNr;
}

File: app/src/main/kotlin/de/domradio/api/data/Onair.kt
package de.domradio.api.data

import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "onair")
data class Onair(
    @PropertyElement(name = "title")
    var title: String? = null,
    @PropertyElement(name = "artist")
    var artist: String? = null,
    @PropertyElement(name = "type")
    var type: String? = null,
    @PropertyElement(name = "presenter")
    var presenter: String? = null,
    @PropertyElement(name = "date")
    var date: String? = null,
    @PropertyElement(name = "duration")
    var duration: String? = null,
    @PropertyElement(name = "start")
    var start: String? = null,
    @PropertyElement(name = "cover")
    var cover: String? = null,
    @PropertyElement(name = "TitelID")
    var TitelID: String? = null,
    @PropertyElement(name = "ElementID")
    var ElementID: String? = null,
    @PropertyElement(name = "ArchivNr")
    var ArchivNr: String? = null
)